doc/nrf/protocols/wifi/wifi.rst

```diff
@@ -0,0 +1,175 @@
+Protocol
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+Wi-Fi® is the branding used by the `Wi-Fi Alliance® <Wi-Fi Alliance_>`_ to describe the family of wireless local area network standards specified by the `IEEE 802.11 Working Group`_.
+The two industry bodies work closely together, with the Wi-Fi Alliance focusing on the requirements and certification of the technology, and the IEEE focusing on the underlying technical specifications.
+The Wi-Fi Alliance is responsible for the Wi-Fi CERTIFIED accreditation program, which aims to achieve high level of interoperability, reliability, and security compliance.
+
+Wi-Fi is an evolving standard, with new IEEE 802.11 standards being incorporated every 4-6 years, along with the corresponding Wi-Fi Alliance certification program.
+The branding *Wi-Fi 6* is defined by the Wi-Fi Alliance, and aligns with the IEEE 802.11ax specification, but includes all previous versions of the IEEE 802.11 specifications dating back to the introduction of Wi-Fi in 1997.
+The evolution from the base standard started with the introduction of IEEE 802.11b and IEEE 802.11a in 1999, IEEE 802.11g in 2003, IEEE 802.11n in 2008 (and subsequently branded Wi-Fi 4), IEEE 802.11ac in 2014 (subsequently branded Wi-Fi 5), and finally IEEE 802.11ax in 2019, branded Wi-Fi 6.
+These standards operate in the unlicensed spectrum of the 2.4 GHz and 5 GHz bands.
+However, 11b and 11g are only applicable in the 2.4 GHz band, while 11a and 11ac are only applicable in the 5 GHz band.
+The IEEE 802.11ax standard is also specified for operation in the 6 GHz band, with this variant being branded Wi-Fi 6E to signify extension to the 2.4 and 5 GHz bands used in Wi-Fi 6.
+Wi-Fi 7, based on the IEEE 802.11be specification, is due to launch by the end of 2023.
+
+For nRF70 series documentation, see the following:
+
+* :ref:`ug_nrf70` provides information related to Wi-Fi support in |NCS| and using the development kit (DK).
+* :ref:`wifi_samples` for the available samples.
+* `Product specification for nRF70 Series devices`_ for the supported features.
+* `Guidelines and application notes for nRF70 Series devices`_.
```

Comment:
Can we move line 8 to line 25 to the main page Wi-Fi so that we can have some generic content about Wi-Fi protocol?